src/app/common/stores/applications/SsvWeb/Account.store.ts

```diff
@@ -63,7 +63,7 @@ class AccountStore extends BaseStore  {
             resolve(res);
         });
         if (result.data) {
-            this.ownerNonce = Number(result.data.nonce) + 1;
+            this.ownerNonce = Number(result.data.nonce);
```

Comment:
Just so you know you could write as +result.data.nonce instead of Number(result.data.nonce)